py/riscv/memory-cva6.py

```diff
@@ -21,4 +21,4 @@ def __init__(self):
 
 
 def configure_memory(interface):
-    interface.addMemoryRange(0, 0x000080000000, 0x0000BFFFFFFF)
+    interface.addMemoryRange(0, 0x000080000000, 0x0000bfffffff)
```

Comment:
This is a very minor detail, which you can decide whether or not you want to address. We now have a net change in the `memory-cva6.py` file that changes `0x0000BFFFFFFF` to `0x0000bfffffff`, i.e. the format is changed but not the value. My preference would be to leave the file unchanged if there aren't any substantive changes to make.